.github/chainguard/lifecycle-automation.sts.yaml

```diff
@@ -0,0 +1,13 @@
+issuer: https://accounts.google.com
+
+# have more than one service account
+subject_pattern: "(123|456)"
```

Comment:
We'll want to fill in real values, but this LGTM

Reply:
ahh, i see, ok

i will collect the unique ids

Reply:
done